bin/vega-cli

```diff
@@ -1,22 +1,31 @@
 #!/usr/bin/env node
 
-var _ = require('lodash');
-var vega = require('vega');
+var _ = require('lodash')
+var vega = require('vega')
+var momentTZ = require('moment-timezone')
+var numeral = require('numeral')
 
-let data = '';
-process.stdin.setEncoding('utf8');
+let data = ''
+process.stdin.setEncoding('utf8')
 
 process.stdin.on('readable', () => {
-  let chunk;
+  let chunk
   while ((chunk = process.stdin.read()) !== null) {
-    data += chunk;
+    data += chunk
   }
-});
+})
+
+vega.expressionFunction('momentFormat', (date, fmt) =>
```

Comment:
That's an optimdata specific function. i think this should be in a vega-cli in an optimdata private repo